openedx_tagging/core/tagging/import_export/api.py

```diff
@@ -124,14 +125,18 @@ def get_last_import_status(taxonomy: Taxonomy) -> TagImportTaskState:
     Get status of the last import task of the given taxonomy
     """
     task = _get_last_import_task(taxonomy)
-    return task.status
+    if task is None:
+        raise ValueError("No import task was created yet.")
+    return TagImportTaskState(task.status)
```

Comment:
This was a bug: the typing declared that it returned enum instances, but it was returning the enum _values_ which are just plain strings. Also it wasn't handling the case where there was no "last import task".